models/gif_cond_exp_multisynapse.cpp

```diff
@@ -127,11 +127,11 @@ nest::gif_cond_exp_multisynapse::Parameters_::Parameters_()
   , V_reset_( -55.0 )  // mV
   , Delta_V_( 0.5 )    // mV
   , V_T_star_( -35 )   // mV
-  , lambda_0_( 0.001 ) // 1/ms
+  , lambda_0_( 1. )    // 1/s
```

Comment:
This must remain in milliseconds, since the value is used as
```
const double lambda = P_.lambda_0_ * std::exp( ( S_.V_ - S_.sfa_ ) / P_.Delta_V_ );
...
-numerics::expm1( -lambda * Time::get_resolution().get_ms() )
```
i.e., it is multiplied by a value in milliseconds.

The user provides/retrieves `lambda_0` in 1/s; the value is converted to the internal units of 1/ms in `Parameters_::get()/set()`.  This is the standard approach in NEST.